package.json

```diff
@@ -87,30 +87,30 @@
     "libnpmsearch": "^6.0.2",
     "libnpmteam": "^5.0.3",
     "libnpmversion": "^4.0.2",
-    "make-fetch-happen": "^11.1.1",
+    "make-fetch-happen": "^13.0.0",
     "minimatch": "^9.0.3",
-    "minipass": "^5.0.0",
+    "minipass": "^7.0.3",
     "minipass-pipeline": "^1.2.4",
     "ms": "^2.1.2",
     "node-gyp": "^9.4.0",
     "nopt": "^7.2.0",
     "npm-audit-report": "^5.0.0",
-    "npm-install-checks": "^6.1.1",
-    "npm-package-arg": "^10.1.0",
-    "npm-pick-manifest": "^8.0.1",
-    "npm-profile": "^7.0.1",
-    "npm-registry-fetch": "^14.0.5",
+    "npm-install-checks": "^6.2.0",
+    "npm-package-arg": "^11.0.0",
+    "npm-pick-manifest": "^9.0.0",
+    "npm-profile": "^9.0.0",
+    "npm-registry-fetch": "^16.0.0",
     "npm-user-validate": "^2.0.0",
     "npmlog": "^7.0.1",
     "p-map": "^4.0.0",
-    "pacote": "^15.2.0",
+    "pacote": "^17.0.1",
     "parse-conflict-json": "^3.0.1",
     "proc-log": "^3.0.0",
     "qrcode-terminal": "^0.12.0",
     "read": "^2.1.0",
     "semver": "^7.5.4",
     "sigstore": "^1.7.0",
```

Comment:
@feelepxyz plz check if this needs an update? i see make-fetch is updated above

Reply:
https://github.com/npm/pacote/pull/303/files

Reply:
This will be updated once pacote@17.0.2 is released https://github.com/npm/pacote/pull/304